package.json

```diff
@@ -48,6 +48,6 @@
     "fs-extra": "^0.11.0",
     "lodash": "~2.4.1",
     "recursive-readdir": "^1.2.0",
-    "titanium": ">3.2.0"
```

Comment:
@FokkeZB unless I'm missing something, it seems to me that there's no reason to remove support for earlier versions of the titanium cli here. If I'm understanding, ti cli 4+ doesn't require a login. Rather than removing the login check and removing support for ti cli 3.x.x, you should probably instead make checks in the code to see what version of the ti cli is installed as a dependency, and then decide whether or not to skip the login check.

In short, there's no sense in removing support for earlier versions of the ti cli if a simple conditional can avoid it, rather then stripping out functionality.